pkg/generate/app/templatelookup.go

```diff
@@ -57,9 +61,13 @@ func (r TemplateSearcher) Search(terms ...string) (ComponentMatches, error) {
 						Score:       score,
 						Template:    template,
 					})
+					break
```

Comment:
i'm confused by this break.  what if the term you "matched" an item to was not an exact match?  don't you want to search on the other terms to see if there is a better match?  (i'm not clear under what conditions we'd actually have multiple terms either, but...)